example/z-docker-opam/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ocaml/opam:alpine as build
+
+# Install system dependencies
+RUN sudo apk add --update libev-dev openssl-dev
+
+WORKDIR /build
+
+# Install dependencies
+ADD hello.opam hello.opam
+RUN eval $(opam env) && opam install . --deps-only
```

Comment:
AFAIK `eval $(opam env)` is not necessary if you are using the `opam` binary. It would have been necessary for `dune build` later, but `opam exec` takes care of the environment there, too.

Reply:
You're right, will change that